zebra-state/src/service/non_finalized_state/chain.rs

```diff
@@ -506,25 +551,22 @@ mod tests {
         proptest!(ProptestConfig::with_cases(env::var("PROPTEST_CASES")
                                           .ok()
                                           .and_then(|v| v.parse().ok())
-                                          .unwrap_or(1)),
-        |((chain, count) in arbitrary_chain_and_count())| {
-            let chain = chain.0;
-            let fork_tip_hash = chain[count - 1].hash();
+                                          .unwrap_or(32)),
```

Comment:
Can we make this into a constant?
```suggestion
                                          .unwrap_or(DEFAULT_PARTIAL_CHAIN_PROPTEST_CASES)),
```

Reply:
Should I also change `PROPTEST_CASES` to const in other tests?
https://github.com/ZcashFoundation/zebra/blob/15843cfd6e046391672e1a9f91fb2bf81aecf3af/zebra-chain/src/work/tests/prop.rs#L50
https://github.com/ZcashFoundation/zebra/blob/15843cfd6e046391672e1a9f91fb2bf81aecf3af/zebra-chain/src/block/tests/prop.rs#L63

Reply:
Sure!

Making a named constant is helpful when there are multiple copies of the same number in a file, so they all get changed at the same time.

Reply:
Updated.